install_files/ansible-base/roles/common/templates/apt-daily-timer-override.j2

```diff
@@ -0,0 +1,5 @@
+[Timer]
+OnCalendar=
+OnCalendar=*-*-* {{ (daily_reboot_time|int - 1) % 24 }}:00
```

Comment:
The goal of this modulus is to ensure that `apt-daily.timer` is configured to run 1h before `apt-daily-upgrade`, so the apt lists are quite new before installing upgrades.

Reply:
This works on the staging instance.

Reply:
Running the update more frequently would increase the chance of applying upgrades as soon as possible. You could update every hour except the one in which upgrade is performed with:
```
OnCalendar=*-*-* {{ range(0,24) | difference([ (daily_reboot_time|int) % 24 ]) | join(",") }}:00
```